packages/framework/src/hooks/useHtmlPassThrough.tsx

```diff
@@ -0,0 +1,29 @@
+import type { RefCallback } from "react";
+import { isEmpty, isString, mapKeys } from "lodash-es";
+import { useCallback } from "react";
+
+export const useHtmlPassThrough = (
+  widgetId: string,
```

Comment:
Let's rename this to testId, make it nullable, and second param.

Reply:
@evshi what you mean by  second param ?